### ISSUE TITLE
LibCal : Background color on monthly hours display is borked

### ISSUE THREAD

**Issue by tomkeays** [tomkeays/lemoyne#86]
Currently displaying text of #777 on a background of #000. Big time accessibility issue, non-compliant with Le Moyne Library styles ... and ugly.

![image](https://user-images.githubusercontent.com/171037/183452581-818ba6f6-ce04-4796-b29e-476bc5aaa214.png)

See Monthly tab of https://lemoyne.libcal.com/hours/

Add this to local stylesheet in [Custom JS/CSS Code](https://lemoyne.libcal.com/admin_look.php?action=0) of Look & Feel to fix.

```
<style>
/* Fix background of Monthly calendar display in https://lemoyne.libcal.com/hours/ */
.s-lc-hm-loc {
  color: #000;
}
.s-lc-hm-bg-662 {
  background-color: inherit;
}
.s-lc-hm-bg-585 {
  background-color: inherit;
}
</style>
```

**Comment by tomkeays** [comment]
For good measure, should probably beef up the font size as well as making the font color black. Like this, maybe?

```
.s-lc-hm-day-l {
  color: #000;
  font-size: 1.5rem;
}
.s-lc-hm-loc {
  color: #000;
  font-size: 1.4rem;
}
```

With the other change, it gives us this:

![image](https://user-images.githubusercontent.com/171037/183452345-3bc97d42-c52f-410a-8986-32ef33cc2775.png)

**Comment by shostap** [comment]
This was done in consultation with Tom. We took care of it last week. The LibCal systems style sheet loads in after the local css so you can't override the system css for somethings like the background in each date cell.
